channeldb/channel.go

```diff
@@ -2528,7 +2528,7 @@ func (c *OpenChannel) LoadFwdPkgs() ([]*FwdPkg, error) {
 	defer c.RUnlock()
 
 	var fwdPkgs []*FwdPkg
-	if err := kvdb.View(c.Db, func(tx kvdb.RTx) error {
+	if err := kvdb.View(c.Db.chanCache, func(tx kvdb.RTx) error {
```

Comment:
Similar comment here re not caching this bucket as it may be very large for long lived larger routing nodes.